demisto_sdk/commands/openapi_codegen/openapi_codegen.py

```diff
@@ -0,0 +1,837 @@
+        for param in re.findall(r'{([^}]+)}', command['path']):  # get content inside curly brackets
+            if param in illegal_function_names:
+                command['path'] = command['path'].replace(param, f'{param}{prepend_illegal}')
+        req_function = req_function.replace('$PATH$', command['path'])
+        if new_params:
+            params = self.format_params(new_params, base_params, '$PARAMS$')
+            req_function = req_function.replace('$PARAMETERS$', params)
+        else:
+            req_function = '\n'.join(
+                [x for x in req_function.split('\n') if '$PARAMETERS$' not in x])
+        if new_data:
+            new_data = self.format_params(new_data, base_data, '$DATAOBJ$')
+            req_function = req_function.replace('$DATA$', new_data)
+        else:
+            req_function = '\n'.join(
+                [x for x in req_function.split('\n') if '$DATA$' not in x])
+        if new_params:
+            req_function = req_function.replace(
+                '$NEWPARAMS$', ', params=params')
+        else:
+            req_function = req_function.replace('$NEWPARAMS$', '')
+        if new_data:
+            req_function = req_function.replace('$NEWDATA$', ', json_data=data')
+        else:
+            req_function = req_function.replace('$NEWDATA$', '')
+        if headers:
+            new_headers = []
+            for header in headers:
+                for k, v in header.items():
+                    new_headers.append(base_header.replace('$HEADERKEY$', f"'{k}'")
+                                       .replace('$HEADERVALUE$', f"'{v}'"))
+
+            req_function = req_function.replace('$HEADERSOBJ$', ' \n        '.join(new_headers))
+        else:
+            req_function = req_function.replace('$HEADERSOBJ$', '')
+        if self.configuration['context_path']:
+            context_name = self.context_path
+        else:
+            context_name = command['name'].title().replace('_', '')
+        if command.get('context_path'):
+            function = function.replace('$CONTEXTPATH$', f'.{command["context_path"]}')
+        else:
+            function = function.replace('$CONTEXTPATH$', '')
+        if command.get('root_object'):
+            function = function.replace('$OUTPUTS$', f"response.get('{command['root_object']}')")
+        else:
+            function = function.replace('$OUTPUTS$', 'response')
+        if command['unique_key']:
+            function = function.replace('$UNIQUEKEY$', command['unique_key'])
+        else:
+            function = function.replace('$UNIQUEKEY$', '')
+        function = function.replace('$CONTEXTNAME$', context_name)
+        return function, req_function
+
+    def get_yaml(self, no_code):
+        # Create the commands section
+        commands = []
+        for command in self.configuration['commands']:
+            args = []
+            for arg in command['arguments']:
+                options = None

```

Comment:
we can remove `auto` and `options` from here

Reply:
why?